### PR TITLE
Upgrade Bitmovin Player dependency

### PR DESCRIPTION
**Pull request by realeyes-craig**
- Upgrade Bitmovin Player to 2.57.1
This also requires these upgrades:
- Minimum deployment target to 10.0
- Bump GoogleAds-IMA-iOS-SDK to 3.12.1

The need for this upgrade is outlined here: #30

### CODE REVIEW

File: BitmovinConvivaAnalytics.podspec [hunk 1]
```diff
@@ -12,16 +12,16 @@ Conviva Analytics Integration for the Bitmovin Player iOS SDK
...
 
   s.source_files = 'BitmovinConvivaAnalytics/Classes/**/*.swift'
   s.resources = 'BitmovinConvivaAnalytics/Assets/*'
   s.swift_version = '4.2'
   s.cocoapods_version = '>= 1.4.0'
 
-  s.ios.dependency 'BitmovinPlayer', '~> 2.51'
-  s.tvos.dependency 'BitmovinPlayer', '~> 2.51'
+  s.ios.dependency 'BitmovinPlayer', '~> 2.57.1'
+  s.tvos.dependency 'BitmovinPlayer', '~> 2.57.1'
```
Comment (stonko1994) [inline BitmovinConvivaAnalytics.podspec:24]: Please update the version in the `README` accordingly as well.

**Review by stonko1994 — COMMENTED** [review]
Please try to avoid adding links to non-public issue management systems as this repository is open source.

File: BitmovinConvivaAnalytics.podspec (hunk 1 above)
Comment (realeyes-craig) [reply BitmovinConvivaAnalytics.podspec:24]: Which line needs to be updated in the `README`?
This PR would not have a version bump so it looks like updating the `Compatibility` section wouldn't be needed until there is a release.

Comment (stonko1994) [reply BitmovinConvivaAnalytics.podspec:24]: I was thinking of something like: (_Even though the links do not exist yet - and assuming `1.3` as the next version_)

---

### Compatibility
This version of the Conviva Analytics Integration depends on `BitmovinPlayer` version `>= 2.57.1`. If you need support for prior `2.57.1` please use [v1.2.0](https://github.com/bitmovin/bitmovin-player-ios-analytics-conviva/tree/1.2.0). For `BitmovinPlayer` version `>= 2.57.1` please use [v1.3.0](https://github.com/bitmovin/bitmovin-player-ios-analytics-conviva/tree/1.3.0) or later.

Comment (realeyes-craig) [reply BitmovinConvivaAnalytics.podspec:24]: I understand. With assuming 1.3 is the next version I updated the text like so:

The most recent version of the Conviva Analytics Integration depends on `BitmovinPlayer` version `>= 2.57.1`.
For `BitmovinPlayer` version `>= 2.57.1` please use [v1.3.0](https://github.com/bitmovin/bitmovin-player-ios-analytics-conviva/tree/1.3.0) or later.

If you need support for prior to `2.57.1` please use [v1.2.0](https://github.com/bitmovin/bitmovin-player-ios-analytics-conviva/tree/1.2.0).

**Review by stonko1994 — APPROVED** [review]

**Comment by luckygoyal-bitmovin** [comment]
Hi @manfredrunkel , can you please review/approve this PR. It will be useful to merge this and create new release tag to align the implementation with newer Bitmovin player versions.

**Review by dweinber — APPROVED** [review]

**Pull request merged**